src/signalrclient/hub_connection_impl.cpp

```diff
@@ -139,8 +143,28 @@ namespace signalr
             .append(message.serialize()));
     }
 
+    bool hub_connection_impl::invoke_callback(const web::json::value& message)
+    {
+        auto is_progress = message.has_field(_XPLATSTR("P"));

```

Comment:
These values are tightly coupled to the protocol. Changing this you would create one more level of indirection which could be even more confusing. I tried using good variable names to make it readable thought. For instance this line `auto is_progress = message.has_field(_XPLATSTR("P"));` says that if a message has "P" field it is a progress message.